internal/rbd/manager.go

```diff
@@ -268,3 +316,108 @@ func (mgr *rbdManager) CreateVolumeGroup(ctx context.Context, name string) (type
 
 	return vg, nil
 }
+
+func (mgr *rbdManager) CreateVolumeGroupSnapshot(
+	ctx context.Context,
+	vg types.VolumeGroup,
+	name string,
+) (types.VolumeGroupSnapshot, error) {
+	// TODO: reserve a unique name for the snapshot
```

Comment:
Need to address the TODO?

Reply:
No, that is actually done. I'll remove the comment.